lib/tidewave.ex

```diff
@@ -10,7 +10,11 @@ defmodule Tidewave do
       allow_remote_access: Keyword.get(opts, :allow_remote_access, false),
       phoenix_endpoint: nil,
       inspect_opts:
-        Keyword.get(opts, :inspect_opts, charlists: :as_lists, limit: 50, pretty: true)
+        Keyword.get(opts, :inspect_opts, charlists: :as_lists, limit: 50, pretty: true),
+      tools: [
+        exclude: Enum.map(opts[:tools][:exclude] || [], &to_string/1),
+        include: opts[:tools][:include] && Enum.map(opts[:tools][:include], &to_string/1)
+      ]
```

Comment:
I would either flatten this or convert it to a nested map:

```suggestion
      tools_exclude: Enum.map(opts[:tools][:exclude] || [], &to_string/1),
      tools_include: opts[:tools][:include] && Enum.map(opts[:tools][:include], &to_string/1)
```

Reply:
To be clear, the public API can be nested, but we should normalize it internally.